### DISCUSSION TITLE
Navier Stokes over multiple blocks

### DISCUSSION THREAD

**Discussion by oh304** [idaholab/moose#27817] [Q&A Modules: Navier-Stokes] [answered]
I was wondering if there was a way to combine temperatures in an auxkernel? I have more than one temperature variable, as a result of using navier stokes, and was wondering if there was a way to combine them into one master variable T so the simulation can be read more easily in paraview. Thanks :)

**Comment by GiudGiud** [comment, marked as answer]
Hello

You can use two ProjectionAux for that.
https://mooseframework.inl.gov/source/auxkernels/ProjectionAux.html

Create a 3rd auxiliary temperature variable like "temp_out" and use two projectionAux to copy T_fluid and T_solid into it.

This is strictly for output purposes. For using in the nonlinear system, we do not recommend lagging variable values / losing derivative information by using the auxiliary system.

Guillaume

**Reply by oh304** [reply]
Thank you!!
